INSTALLATION.md

```diff
@@ -1,8 +1,10 @@
-# Installation
+# Quick-Start
 
-We have two different ways to experience Pouch, one for end-users, the other for developers. In the former one, installation can help you with Pouch installing out of box. The latter one is for those who wish to get involved in Pouch with hacking.
+Two quick-start are provided, one for end-users, the other for developers.
```

Comment:
Do we need to use `Two quick starts` instead of `Two quick start`? Just double check.

Reply:
will fix.